setup.py

```diff
@@ -17,27 +17,17 @@
 #
 
 
-import sys
-
-if sys.version_info < (3, 6):
-    # edgedb.NamedTuple() relies on ordered **kwargs.
-    raise RuntimeError('edgedb requires Python 3.6 or greater')
-
 import os
-import os.path
 import pathlib
 import re
 import subprocess
-
-# We use vanilla build_ext, to avoid importing Cython via
```

Comment:
`distutils` is deprecated and will be removed in next version https://www.python.org/dev/peps/pep-0632/